analysis/deep2_r23_o32/calibration_plots.py

```diff
@@ -24,25 +24,39 @@ def lac_gpc_plots(fitspath, fitspath_ini, dataset, raw=False, apply_dust=False,
     Outputs
     pdf_files
     """
+
+    # Call for validation table
     if revised:
-        temp_table = asc.read(join(fitspath, filename_dict['bin_derived_prop_rev_dust']))
+        rev_s = ''
         verification = asc.read(join(fitspath, filename_dict['bin_valid_rev']))
-        pea_out_pdf = join(fitspath, dataset+'_GPC.revised.pdf')
-        LAC_out_pdf = join(fitspath, dataset+'_LAC.revised.pdf')
     else:
-        temp_table = asc.read(join(fitspath, filename_dict['bin_derived_prop']))
+        rev_s = '.valid1'
         verification = asc.read(join(fitspath, filename_dict['bin_valid']))
-        pea_out_pdf = join(fitspath, dataset+'_GPC.pdf')
-        LAC_out_pdf = join(fitspath, dataset+'_LAC.pdf')
+
+    if raw:
+        mc = '.MC'
+    else:
+        mc = ''
+
+    if apply_dust:
+        ad = '.dustcorr'
+    else:
+        ad = ''
+    tempature_table = fitspath + 'bin_derived_properties' + rev_s + mc + ad + '.tbl'
+    pea_out_pdf = fitspath + dataset + '_GPC' + rev_s + mc + ad + '.pdf'
+    LAC_out_pdf = fitspath + dataset + '_LAC' + rev_s + mc + ad + '.pdf'
```

Comment:
Avoid defining here. This breaks up `temperature_table` and `temp_table` making it hard to find related lines

Reply:
applied